Site/ASAB Maestro/Files/grafana/provisioning/datasources.yaml

```diff
@@ -0,0 +1,88 @@
+# config file version
+apiVersion: 1
+
+# list of datasources that should be deleted from the database
+deleteDatasources:
+  - name: InfluxDB
+    orgId: 1
+  - name: InfluxDBTelegraf
+    orgId: 1
+
+# list of datasources to insert/update depending
+# whats available in the database
+datasources:
+  # <string, required> name of the datasource. Required
+- name: InfluxDB

```

Comment:
Must be aligned with https://ndocs.teskalabs.com/logman.io/metrics/viewing-metrics/ 

... which _mainly_ means that all telemetry is in "InfluxDB" datasource.

Which is wrong too, b/c it names the technology.
But ... I give up that fight here.